### PR TITLE
Add detection for non-wechat user

### PR DESCRIPTION
**Pull request by kmahyyg**
**NOT TESTED WITH JEKYLL**
Tested using WebStorm with Chrome, it should not have any display problem.
Already checked using the different device view in Chrome Dev Tools.

qrcode.html could still be optimized to load much faster. (cuz it's generated with markdown on Typora)

The UA check is tested with Chrome UA change Extension.

If you met any problem, plz let me know ASAP, so that I could improve myself. Thanks.

### CODE REVIEW

**Comment by kmahyyg** [comment]
Also, this UA checker just divided our users to three groups:

- Mobile Wechat user
- Wechat on Windows user
- Non-Wechat user

I hate the CNZZ analytics javascript and also Baidu's. **BECAUSE IT USED TO BE HIJACKED BY CHINA GOVERNMENT TO ATTACK GITHUB SEVERAL YEARS AGO.**

**Comment by kmahyyg** [comment]
add busuanzi analytics, pure and fast with no configuration.

http://ibruce.info/2015/04/04/busuanzi/

**Comment by kmahyyg** [comment]
If you think this analytics is not enough, please consider using leancloud service.
You may need to do some coding.

For more details and sdk document, see:

- https://leancloud.cn/docs/js_analytics.html
- http://www.jeyzhang.com/hexo-next-add-post-views.html

File: index.html [hunk 1]
```diff
@@ -89,24 +89,11 @@ <h5 class="card-title">二维码</h5>
...
-        (function() {
-            var hm = document.createElement("script");
-            hm.src = "https://hm.baidu.com/hm.js?fd1e5514c663d2dcb07f636e1ae88da0";
-            var s = document.getElementsByTagName("script")[0]; 
-            s.parentNode.insertBefore(hm, s);
-        })();
-    </script>
-    -->
-    
-    <script src="https://s19.cnzz.com/z_stat.php?id=1273862125&web_id=1273862125" language="JavaScript"></script>
```
Comment (thiswind) [inline index.html:108]: but it's my tracking id....

Comment (kmahyyg) [reply index.html]: never mind. Those service which may be abused by china government should be taken down.

Comment (kmahyyg) [reply index.html]: so i provided a replacement using busuanzi or leancloud

**Pull request closed without merging**